go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/mariocandela/beelzebub/v3
 
-go 1.20
+go 1.23.0
```

Comment:
Why into continuous integration you use 1.24.1 ? But into go.mod 1.23.0 ? It's a typo ? 😄